Game/MeadowRemixOptions.cs

```diff
@@ -108,10 +110,18 @@ public override void Initialize()
                 new OpLabel(10f, 400, "[Experimental Features]", bigText: true),
                 new OpLabel(10f, 380, "WARNING: Experimental features may cause data corruption, back up your saves", bigText: false),
 
-                new OpLabel(10f, 350, "Custom Story Slugcat", bigText: false),
+                new OpLabel(10f, 350, "Custom Story Slugcat:", bigText: false),
 
-                new OpCheckBox(SlugcatCustomToggle, new Vector2(10f, 320)),
-                new OpLabel(40f, 320, RWCustom.Custom.ReplaceLineDelimeters("If selected, hosts can choose slugcat campaigns that are unstable. <LINE>Clients can choose their own Slugcats inside a host's Story campaign"))
+                new OpCheckBox(SlugcatCustomToggle, new Vector2(160f, 350)),
+
+                new OpLabel(40f, 320, RWCustom.Custom.ReplaceLineDelimeters("If selected, hosts can choose slugcat campaigns that are unstable."))
+                {
+                    verticalAlignment = OpLabel.LabelVAlignment.Center
+                },
+
+                new OpLabel(10f, 290, "Require Campaign Slugcat:", bigText: false),
```

Comment:
Please remove traces of remix bool due to move in story menu

Reply:
This should be resolved now.